Nodejs/Product/Nodejs/PkgCmdId.cs

```diff
@@ -24,8 +24,12 @@ internal static class PkgCmdId
         public const int cmdidNpmOpenModuleHomepage = 0x306;
         public const int menuIdNpm = 0x3000;
 
-        public const int cmdidWorkSpaceNpmInstallMissing = 0x4001;
-        public const int cmdidWorkSpaceNpmInstallNew = 0x4002;
-        public const int cmdidWorkSpaceNpmUpdate = 0x4003;
+        public const int cmdidWorkSpaceNpmInstallMissing = 0x0200;
```

Comment:
>0x0200 [](start = 59, length = 6)

This won't collide with `cmdidReplWindow` because it's in a submenu?

Reply:
No, they only to be unique per group.